src/store/SignUpForm/reducer.js

```diff
@@ -0,0 +1,42 @@
+import {INPUT_NAME, INPUT_EMAIL, INPUT_PASSWORD} from "./constants";
+
+const initState = {
+    name: undefined,
+    email: undefined,
+    password: undefined,
+};
+
+export default function reduce(state = initState, action) {
+    switch (action.type) {
+        case INPUT_NAME:
+            return {...state, name: action.name};
+        case INPUT_EMAIL:
+            return {...state, email: action.email};
+        case INPUT_PASSWORD:
+            return {...state, password: action.password};
+        default:
+            return state;
+    }
+}
+
+/**
+ * Name of signUp user
+ * @param state
+ * @returns {string}
+ */
+export function getName(state) {
```

Comment:
I don’t understand what is the point of creating separate get and set methods, this is more of a java approach. Redux does it himself

Reply:
delete it